share/SEDs/getSEDs.py

```diff
@@ -16,23 +16,27 @@
 #    and/or other materials provided with the distribution.
 #
 """@file getSEDs.py
-Grab example SEDs from the web, clip them at 22050 Angstroms, and then thin with rel_err = 1.e-5 or
-1.e-3 (two versions).
-Note that the outputs of this script, which are the files GALSIM_DIR/share/CWW_*.sed, are
+Grab example SEDs from the web, clip them and then thin.
+Note that the outputs of this script, which are the files GALSIM_DIR/share/SEDs/CWW_*.sed, are
```

Comment:
I guess I'd rather include a README.txt instead of these scripts in the share directory and just point to the scripts in GalSim/devel/ (or some subdirectory).  The README should probably just say the URL where we got each of the kinds of files and what thinning parameters we used on them.  That seems more useful to an end user trying to understand provenance than parsing the scripts (simple though they are).